include/nlohmann/detail/input/input_adapters.hpp

```diff
@@ -45,6 +46,27 @@ struct input_adapter_protocol
 /// a type to simplify interfaces
 using input_adapter_t = std::shared_ptr<input_adapter_protocol>;
 
+/*!
+Input adapter for stdio file access. This adapter read only 1 byte and do not use any
+ buffer. This adapter is a very low level adapter. This adapter
+*/
```

Comment:
This comment is incomplete. Please update the text here.

Reply:
updated as per your comment